pkg/subctl/cmd/gather.go

```diff
@@ -82,23 +85,43 @@ func gatherData() {
 	err := checkGatherArguments()
 	exitOnError("Invalid arguments", err)
 
-	for module, ok := range gatherModuleFlags {
-		if ok {
-			for dataType, ok := range gatherTypeFlags {
-				if ok {
-					status := cli.NewStatus()
-					status.Start(fmt.Sprintf("Gathering %s %s...", module, dataType))
-					status.End(cli.CheckForError(gatherFuncs[module](status, dataType)))
+	configs, err := getMultipleRestConfigs(kubeConfig, kubeContext)
+	exitOnError("Error getting REST config", err)
+	for _, item := range configs {
+		clientSet, err := kubernetes.NewForConfig(item.config)
+		exitOnError("error getting clientset", err)
+		clustername := item.clusterName
+		fmt.Println("Gathering information from", clustername)
+		for module, ok := range gatherModuleFlags {
+			if ok {
+				for dataType, ok := range gatherTypeFlags {
+					if ok {
+						status := cli.NewStatus()
+						status.Start(fmt.Sprintf("Gathering %s %s", module, dataType))
+						status.End(cli.CheckForError(gatherFuncs[module](status, clientSet, dataType, clustername)))
+					}
 				}
 			}
 		}
+		fmt.Println("")
 	}
 }
 
-func gatherConnectivity(status *cli.Status, dataType string) error {
+func gatherConnectivity(status *cli.Status, clientSet kubernetes.Interface, dataType string, clustername string) error {
 	switch dataType {
 	case Logs:
-		status.QueueWarningMessage("Gather Connectivity Logs not implemented yet")
+		err := gather.GetGatewayPodsLogs(clientSet, clustername)
+		if err != nil {
+			status.QueueFailureMessage(fmt.Sprintf("Gathering Gateway pods logs not successful %s", err))
+		} else {
+			status.QueueSuccessMessage("Successfully collected Gateway pods logs")
```

Comment:
This applies to all other occurrences of "_pods logs_" as well. Also let's be consistent and use the term _gather_ everywhere. 

```suggestion
			status.QueueSuccessMessage("Successfully gathered Gateway pod logs")
```